components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/utils/RestApiPublisherUtils.java

```diff
@@ -276,9 +276,18 @@ public static File exportOperationPolicyData(OperationPolicyData policyData)
             CommonUtil.createDirectory(archivePath);
             String policyName = archivePath + File.separator + policyData.getSpecification().getName();
             if (policyData.getSpecification() != null) {
-                CommonUtil.writeDtoToFile(policyName, ExportFormat.YAML,
-                        ImportExportConstants.TYPE_POLICY_SPECIFICATION,
-                        policyData.getSpecification());
+                if (format == null || format == "") {
```

Comment:
How about without making this mandatory, shall we have a default format? Similar to what we do with export API format? default is yaml.

Reply:
@wasuradananjith WDYT?

Reply:
Yeah, +1 for that.

Let's add the check here [1] (which we did in export APIs) to [2]. Then, we won't need the above null (empty) check.

[1] https://github.com/wso2/carbon-apimgt/blob/9b7d306a68cf3d2a88cdf7467b54d2cbff6c80bf/components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/impl/ApisApiServiceImpl.java#L3645
[2] https://github.com/wso2/carbon-apimgt/pull/11325/files/43bfad0f3aded00ebffbfaaa60837530415d40d8#diff-7dd9a559374f8084512bf6bd20e8605b080e84b983272b26da1f7b0c5bb7969fR393